packages/integration/CHANGELOG.md

```diff
@@ -77,3 +77,7 @@ Fixed Minor UI Issues
 # 1.6.0
 
 Added support for web platform
+
+# 1.6.1
+
+Fixed getInitials method
```

Comment:
Let's make this "Fixed a bug in method responsible for extracting initials for LMFeedProfilePicture"